docs/open-source-project/release-notes/tizen-3-0-m2.md

```diff
@@ -535,7 +535,7 @@ As a common feature, multi-user support has been added.
 - Major open-source component upgrades and changes
   - Curl has been upgraded from 7.40 to 7.50.2 for stability.
   - Dnsmasq has been upgraded from 2.57 to 2.74 for stability.
-  - wpa_supplicant has been upgraded from 2.4 to 2.5 for the d-bus interface.
+  - wpa_supplicant has been upgraded from 2.4 to 2.5 for the D-Bus interface.
   - libidn has been upgraded from 1.15 to 1.16 for stability.
   - Libcap has been upgraded from 2.21 to 2.24 for stability.
   - Gnutls has been upgraded from 3.3.5 to 3.4.11 for stability.
```

Comment:
On line 595, we have "gdbus" -> GDBus

On line 745, we have "gdbus" -> GDBus

Reply:
I can see that I have already made the changes, I even checked the output to confirm.